packages/ui/src/components/organisms/SKUMatrix/styles.scss

```
@@ -0,0 +1,162 @@
+
+  // Radius
+  --fs-sku-matrix-sidebar-button-radius-pill			: var(--fs-border-radius-pill);
+
+  // Partial
+  --fs-sku-matrix-slide-over-partial-gap			: calc(2 * var(--fs-grid-padding));
+  --fs-sku-matrix-slide-over-partial-width-mobile		: calc(100vw - var(--fs-sku-matrix-slide-over-partial-gap));
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  overflow: auto;
+
+  > * {
+    flex-shrink: 0;
+    padding: var(--fs-sku-matrix-sidebar-spacing-8);
+    padding-bottom: 0;
+  }
+
+  &[data-fs-slide-over] {
+    background-color: var(--fs-sku-matrix-sidebar-bkg-color);
+
+    [data-fs-slide-over-header] {
+      padding: var(--fs-sku-matrix-sidebar-spacing-8);
+      padding-bottom: 0;
+    }
+
+    &[data-fs-slide-over-size="partial"] {
+      width: var(--fs-sku-matrix-slide-over-partial-width-mobile);
+
+      @include media(">=notebook") {
+        max-width: var(--fs-sku-matrix-slide-over-partial-width-mobile);
+      }
+    }
+  }
+
+  [data-fs-sku-matrix-sidebar-title] {
+    font-size: var(--fs-sku-matrix-sidebar-title-size);
+    font-weight: var(--fs-sku-matrix-sidebar-title-text-weight);
+    line-height: 1.12;
+  }
+
+  [data-fs-table] {
+    color: var(--fs-color-neutral-6);
+
+    [data-fs-table-cell] {
+      &:first-child {
+        padding-left: 0;
+      }
+
+      &:last-child {
+        padding-right: 0;
+      }
+    }
+  }
+
+  [data-fs-table-head] {
+    [data-fs-table-cell] {
+      font-size: var(--fs-sku-matrix-sidebar-table-cell-font-size);
+      font-weight: var(--fs-sku-matrix-sidebar-table-cell-text-weight);
+    }
+  }
+
+  [data-fs-table-cell],
+  [data-fs-price-variant="spot"] {
+    font-weight: var(--fs-sku-matrix-sidebar-text-weight-medium);
+  }
+
+  [data-fs-table-cell]:first-child,
+  [data-fs-sku-matrix-sidebar-table-price] {
+    color: var(--fs-color-text);
+  }
+
+  [data-fs-sku-matrix-sidebar-table-price] {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  [data-fs-sku-matrix-sidebar-table-cell-quantity-selector] {
+    width: 1%;
+  }
+
+  [data-fs-sku-matrix-sidebar-cell-image] {
+    display: flex;
+    align-items: center;
+    gap: var(--fs-sku-matrix-sidebar-spacing-2);
+
+    > div {
+      img {
+        object-fit: cover;
+        object-position: center;
+      }
+    }
+  }
+
+  [data-fs-quantity-selector] {
+    [data-fs-icon] {
+      margin: 0;
+    }
+  }
+
+  [data-fs-sku-matrix-sidebar-footer] {
+    display: flex;
+    justify-content: space-between;
+
+    position: sticky;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin-top: auto;
+
+    background-color: var(--fs-sku-matrix-sidebar-bkg-color);
+
+    padding: var(--fs-sku-matrix-sidebar-spacing-4)
+      var(--fs-sku-matrix-sidebar-spacing-8);
+    border-top: 1px solid --fs-color-neutral-2;
```

```suggestion
    border-top: var(--fs-border-width) solid var(--fs-border-color-light);
```

todo: 😬 this border was not being displayed in the footer